README.md

```
@@ -5,6 +5,172 @@
+azdocli project MyProject
+
+# List repositories
+azdocli repos list
+
+# List pipelines
+azdocli pipelines list
+```
+
+### All Subcommands
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)